2.7/alpine/Dockerfile

```diff
@@ -89,6 +67,39 @@ RUN set -ex \
 	)" \
 	&& apk add --virtual .python-rundeps $runDeps \
 	&& apk del .build-deps \
-	&& rm -rf /usr/src/python ~/.cache
+	\
+	&& find /usr/local -depth \
+		\( \
+			\( -type d -a -name test -o -name tests \) \
+			-o \
+			\( -type f -a -name '*.pyc' -o -name '*.pyo' \) \
+		\) -exec rm -rf '{}' + \
+	&& rm -rf /usr/src/python
+
+# if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
+ENV PYTHON_PIP_VERSION 9.0.1
+
+RUN set -ex; \
+	\
+	apk add --no-cache --virtual .fetch-deps openssl; \
+	\
+	wget -O get-pip.py 'https://bootstrap.pypa.io/get-pip.py'; \
+	\
+	apk del .fetch-deps; \
```

Comment:
Yeah, it's a bit odd, but it does help with the cross-variant diffs (doing `vimdiff` across all the different variants in this repo gets pretty tedious, so any little bit helps). :innocent: :+1:

Switching to semicolons is something I thought about doing to the whole file, but decided to keep the change smaller for now.  We've been trying to switch most things over to semicolons (slowly), especially after having fun with things like `foo && for xxx; do something-that-might-fail; done && bar`, which creates a "complex expression", and thus the rules for failure change (such that the `for xxx; do ...; done` bit of that expression will only fail if the final iteration fails, for example).

From https://www.gnu.org/software/bash/manual/bashref.html#index-set:

> The shell does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test in an if statement, part of any command executed in a && or || list except the command following the final && or ||, any command in a pipeline but the last, or if the command’s return status is being inverted with !.

Thanks for taking a look :smile: